### ISSUE TITLE
[BUG] PHP warnings when editing zone configuration

### ISSUE THREAD

**Issue by imyller** [WillyXJ/facileManager#630]
fM Version : **4.6.2**
fmDNS Version : **6.0.5**

Debian 12 (Bookworm)
PHP 8.2.18
Apache 2.4.59

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [X] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [X] I have checked that the issue I'm posting isn't already reported.
- [X] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [X] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---

**(BUG | ISSUE) Expected Behavior:**

Zone edit works without PHP errors and warnings.

**(BUG | ISSUE) Actual Behavior:**

`Warning: Array to string conversion in /opt/facileManager/server/fm-modules/fmDNS/classes/class_zones.php on line 1280

Warning: Undefined property: stdClass::$domain_id in /opt/facileManager/server/fm-modules/fmDNS/classes/class_zones.php on line 1285

Warning: Undefined property: stdClass::$domain_name in /opt/facileManager/server/fm-modules/fmDNS/classes/class_zones.php on line 1286

Warning: Undefined property: stdClass::$domain_mapping in /opt/facileManager/server/fm-modules/fmDNS/classes/class_zones.php on line 1287

Warning: Undefined property: stdClass::$domain_mapping in /opt/facileManager/server/fm-modules/fmDNS/classes/class_zones.php on line 1293

Deprecated: str_replace(): Passing null to parameter #2 ($replace) of type array|string is deprecated in /opt/facileManager/server/fm-modules/fmDNS/classes/class_zones.php on line 1294

Warning: Undefined property: stdClass::$record_count in /opt/facileManager/server/fm-modules/fmDNS/classes/class_zones.php on line 1302

Warning: Undefined property: stdClass::$domain_view in /opt/facileManager/server/fm-modules/fmDNS/classes/class_zones.php on line 1307`

Click zone 'Edit':

<img width="1013" alt="editerr2" src="https://github.com/WillyXJ/facileManager/assets/1898870/e2c4798b-c0c4-4ebf-b035-2a751bd24f97">

Results in:

<img width="1147" alt="editerrr4" src="https://github.com/WillyXJ/facileManager/assets/1898870/9e8116aa-eb61-4f27-a134-5f73308b3042">

**(BUG | ISSUE) Steps to reproduce:**

* Fresh installation
* Create server configuration
* Edit reverse zone (click 'Edit')
* Fatal PHP error is displayed

**Comment by WillyXJ** [comment]
This is now fixed in [fmDNS 6.0.6](https://www.facilemanager.com/modules/fmdns/) and later.

**Comment by imyller** [comment]
Fix caused regression:

<img width="848" alt="701phperr" src="https://github.com/WillyXJ/facileManager/assets/1898870/6be1ef36-596f-46da-b45d-4179b3f67968">

```
Warning: Array to string conversion in /opt/facileManager/server/fm-modules/fmDNS/functions.php on line 701
```

in functions.php `getZoneServers` function:
```
return (string) $serial_no;
```
